packages/dev/docs/src/template.mdx

```diff
@@ -20,6 +20,7 @@ import {Component} from '@react-spectrum/COMPONENT_NAME';
 
 ---
 category: Categoryname
+keywords: []
```

Comment:
template?

Reply:
IMO leaving it empty makes it less error prone than pre-filling it (someone might forget to update it)